auth-web/src/components/auth/account-settings/team-management/MemberDataTable.vue

```diff
@@ -478,6 +500,17 @@ export default class MemberDataTable extends Vue {
     )
   }
 
+  private isBceidOrBcscAccount (): boolean {
```

Comment:
can it be a generic function name like isRegularAccount() ? we never know how IDP would change over time.

Reply:
Sure.. Just rename the method name or u mean dont check using accessType?

Reply:
just the name, as it will be easy to change a code block within a function without impacting anywhere else.

Reply:
If we have a generic function, then we can have a getter as well and use it in template :)